src/validation.cpp

```diff
@@ -1515,7 +1527,7 @@ MempoolAcceptResult MemPoolAccept::AcceptSingleTransaction(const CTransactionRef
                                         effective_feerate, single_wtxid);
 }
 
-PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::vector<CTransactionRef>& txns, ATMPArgs& args)
+PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactionsInternal(const std::vector<CTransactionRef>& txns, ATMPArgs& args)
```

Comment:
In commit "Add transactions to txgraph, but without cluster dependencies"

Nit: add an `AssertLockHeld(m_pool.cs);` here too?